common/tier4_perception_rviz_plugin/src/tools/pedestrian_pose.cpp

```diff
@@ -165,6 +340,108 @@ void PedestrianInitialPoseTool::onPoseSet(double x, double y, double theta)
   dummy_object_info_pub_->publish(output_msg);
 }
 
+void PedestrianInitialPoseTool::publishObjectMsg(
+  const std::array<uint8_t, 16> & uuid, const uint32_t action)
+{
+  Object output_msg;
+  std::string fixed_frame = context_->getFixedFrame().toStdString();
+
+  // header
+  output_msg.header.frame_id = fixed_frame;
+  output_msg.header.stamp = clock_->now();
+
+  // action
+  output_msg.action = action;
+
+  // id
+  output_msg.id.uuid = uuid;
+
+  if (action == Object::DELETE) {
+    dummy_object_info_pub_->publish(output_msg);
+    return;
+  }
+
+  const auto object_tf = objects_.transform(uuid);
+  const auto object_twist = objects_.twist(uuid);
+
+  if (!object_tf || !object_twist) {
+    return;
+  }
+
+  // semantic
+  output_msg.classification.label = ObjectClassification::PEDESTRIAN;
+  output_msg.classification.probability = 1.0;
+
+  // shape
+  output_msg.shape.type = Shape::CYLINDER;
+  const double width = 0.8;
+  const double length = 0.8;
+  output_msg.shape.dimensions.x = length;
+  output_msg.shape.dimensions.y = width;
+  output_msg.shape.dimensions.z = 2.0;
+
+  // initial state
+  // pose
+  tf2::toMsg(object_tf.get(), output_msg.initial_state.pose_covariance.pose);
+  output_msg.initial_state.pose_covariance.pose.position.z = position_z_->getFloat();
+  output_msg.initial_state.pose_covariance.covariance[0] =
+    std_dev_x_->getFloat() * std_dev_x_->getFloat();
+  output_msg.initial_state.pose_covariance.covariance[7] =
+    std_dev_y_->getFloat() * std_dev_y_->getFloat();
+  output_msg.initial_state.pose_covariance.covariance[14] =
+    std_dev_z_->getFloat() * std_dev_z_->getFloat();
+  output_msg.initial_state.pose_covariance.covariance[35] =
+    std_dev_theta_->getFloat() * std_dev_theta_->getFloat();
+  // twist
+  output_msg.initial_state.twist_covariance.twist = object_twist.get();
+
+  dummy_object_info_pub_->publish(output_msg);
+}
+
+int PedestrianInitialPoseTool::processMouseEvent(rviz_common::ViewportMouseEvent & event)
+{
+  if (!enable_interactive_property_->getBool()) {
+    return PoseTool::processMouseEvent(event);
+  }
+
+  if (event.rightDown()) {
+    const auto point = get_point_from_mouse(event);
+    if (point) {
+      if (event.shift()) {
+        const auto uuid = objects_.create(point.value());
+        publishObjectMsg(uuid.get(), Object::ADD);
+      } else if (event.alt()) {
+        const auto uuid = objects_.remove(point.value());
+        publishObjectMsg(uuid.get(), Object::DELETE);
+      } else {
+        const auto uuid = objects_.select(point.value());
```

Comment:
This uuid is not used.

Reply:
fix in https://github.com/autowarefoundation/autoware.universe/pull/534/commits/133f3f7726711f5ef8222d6b18186011deefa3f7